src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/D8FormToCaseTransformerTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service;
+
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.OcrDataField;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.bulkscan.transformation.in.ExceptionRecord;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformer.D8FormToCaseTransformer;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+public class D8FormToCaseTransformerTest {
```

Comment:
I'm ok with this, but it looks a bit different from Divorce, doesn't it? Maybe I'm imagining it.
If it does, why does it?

Reply:
Renamed both class and test to `FRFormToCaseTransformer`